crates/flox-rust-sdk/src/models/floxmetav2/mod.rs

```diff
@@ -67,6 +81,31 @@ impl FloxmetaV2 {
     }
 }
 
+/// Returns the git options for interacting with floxmeta repositories
+// todo: move floxhub host and token to Flox, or integrate config...
+fn floxmeta_git_options(floxhub_host: &str, floxhub_token: &str) -> GitCommandOptions {
+    let mut options = GitCommandOptions::default();
+
+    // set the user config
+    // todo: eventually use the user's name and email once integrated with floxhub
+    options.add_config_flag("user.name", "Flox User");
+    options.add_config_flag("user.email", "floxuser@example.invalid");
+
+    // unset the global and system config
+    options.add_env_var("GIT_CONFIG_GLOBAL", "/dev/null");
+    options.add_env_var("GIT_CONFIG_SYSTEM", "/dev/null");
+
+    // Set authentication with the floxhub token using an inline credential helper.
+    // The credential helper should help avoinding a leak of the token in the process list.
+    options.add_env_var("FLOX_FLOXHUB_TOKEN", floxhub_token);
+    options.add_config_flag(
+        &format!("credentials.{floxhub_host}.helper"),
+        "!f(){ echo ${FLOX_FLOXHUB_TOKEN}; }; f",
```

Comment:
If we don't want to bubble down the token we could make this 

```suggestion
        "!f(){ $out/bin/flox auth token }; f",
```

but I think the error messages git would give us are worse than managing the token ourselves.

Reply:
Current implementation looks fine to me!